document/js-api/index.bs

```diff
@@ -1781,7 +1781,8 @@ In practice, an implementation may run out of resources for valid modules below
 
 <ul>
 <li>The maximum size of a table is 10,000,000.</li>
-<li>The maximum number of pages of a memory is 65,536.</li>
+<li>The maximum size of a 32-bit memory is 65,536 pages (4 GiB).</li>
```

Comment:
Is this actually necessary, given that more than 4G isn't even addressable?

Reply:
Do you think we should just drop this line then?

Reply:
I think its somewhat useful to have the two limits listed next to each other like this.. even though I guess its redundant.

Reply:
Oh nevermind, the limit itself is pre-existing.